doc/index.qdoc

```diff
@@ -95,8 +95,27 @@ CONFIG += link_pkgconfig
 PKGCONFIG += amberwebauthorization
 \endcode
 
-It may then \c{#include <oauth1.h>}, \c{#include <oauth2.h>},
-\c{#include <redirectlistener.h>} as required.
+If you're using a yaml file to package your RPM you'll also need to add
+\c{amberwebauthorization} to the \c{PkgConfigBR} section, like this:
+
+\code
+PkgConfigBR:
+  - amberwebauthorization
+\endcode
+
+Alternatively if you're writing your spec file by hand, add the following to it:
```

Comment:
Starting from SDK 3.10, we are fading out YAML. I would consider switching the order of the paragraphs so that SPEC file syntax would be described first.

Reply:
Thanks for highlighting the phasing out of the yaml files; it's good to emphasise this.

I had a go at rewording it, and with the reordering I struggled to avoid a clunky construction. See the examples below. My feeling is it might be better to keep the same order, but emphasise what you say in the text (and maybe come back and change this after the 3.10 release).

Version 1:
```
You'll also need to add the following to your spec file.

\code
BuildRequires:  pkgconfig(amberwebauthorization)
\endcode

If your project is using a yaml file to generate your spec file, which is
the default prior to Sailfish SDK 3.10, don't edit your spec file, but
instead amend the yaml file to add \c{amberwebauthorization} to the
\c{PkgConfigBR} section, like this:

\code
PkgConfigBR:
  - amberwebauthorization
\endcode
```
Version 2:
```
If you're using a yaml file to generate your spec file you'll need to add
\c{amberwebauthorization} to the \c{PkgConfigBR} section, like this:

\code
PkgConfigBR:
  - amberwebauthorization
\endcode

Use of yaml files is being phased out from SDK 3.10. After that you should
edit your spec file directly:

\code
BuildRequires:  pkgconfig(amberwebauthorization)
\endcode
```

Reply:
I don't think we need to emphasize the SDK change here. The YAML files keep on working in the future as well, and if someone is using them, there's no need for the developer to stop using them. And even if there was (a need), this is not the correct place to notify the developers about the change. I merely suggested the change in the order so that someone already using SPEC files would not be tempted to start using YAML files instead.

Reply:
So I would prefer this: 
```
You'll also need to add the following to your RPM SPEC file.

\code
BuildRequires:  pkgconfig(amberwebauthorization)
\endcode

If your project is using a YAML file to generate your SPEC file, don't edit your SPEC file, but
instead amend the YAML file to add \c{amberwebauthorization} to the
\c{PkgConfigBR} section, like this:

\code
PkgConfigBR:
  - amberwebauthorization
\endcode
```

Reply:
Done.